index.js

```diff
@@ -42,7 +58,7 @@ name.encode = function (str, buf, offset) {
 
 name.encode.bytes = 0
 
-name.decode = function (buf, offset) {
+name.decode = function (buf, offset, mail = false) {
```

Comment:
```suggestion
name.decode = function (buf, offset, {mail = false} = {}) {
```

Let's use an option object to avoid the [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap).

Reply:
Thx for the review! I added mail now as option object.

Reply:
Looks good.